util/timer/src/timer.rs

```diff
@@ -78,11 +80,13 @@ impl TimerLoop {
     where
         T: 'static + TimeoutHandler, {
         let mut timers = self.timers.write();
-        match timers.entry(name) {
+        let timer_id = self.timer_id_nonce.fetch_add(1, Ordering::SeqCst);
```

Comment:
We still need a timer name to create a new timer.
What's the reason of this change?

Reply:
I've changed it for future. `NetworkExtension` refactor will require `TimerApi` be created without its name. The name is still used for logging in later commit.